tensorflow/lite/experimental/swift/BUILD.apple

```diff
@@ -25,22 +35,57 @@ package(
 
 swift_library(
```

Comment:
We may want to add a comment stating that by default this target builds the core TensorFlow Lite Swift library and that developers will need to specify the appropriate defines to enable delegation support.

Reply:
Moved the other comment up here since it makes more sense to be with this I think